python/lsst/afw/cameraGeom/cameraFactory.py

```diff
@@ -94,10 +95,11 @@ def makeTransformDict(transformConfigDict):
     if transformConfigDict is not None:
         for key in transformConfigDict:
             transform = transformConfigDict[key].transform.apply()
-            resMap[CameraSys(key)] =  transform
+            resMap[CameraSys(key)] = transform
```

Comment:
This is not an issue here, but it's encouraged that whitespace only changes go in a separate commit.  Obviously, use common sense, but I wanted to mention it.